modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/FilePageStore.java

```diff
@@ -291,8 +292,6 @@ private long checkFile(FileIO fileIO, File cfgFile) throws IOException {
 
             fileIO = null;
 
-            if (delete)
-                Files.delete(pathProvider.apply());
```

Comment:
Can you please explain why you removed these two lines?

Reply:
mistake. Reverted